src/par/test/cluster.py

```diff
@@ -0,0 +1,16 @@
+from openroad import Design, Tech
+import helpers
+import par_aux
+
+tech = Tech()
+tech.readLef("Nangate45/Nangate45.lef")
+tech.readLiberty("Nangate45/Nangate45_typ.lib")
+design = Design(tech)
+
+design.readVerilog("gcd.v")
+design.evalTclString("link_design gcd")
```

Comment:
We should add this to the python api as it is a common operation

Reply:
You mean "link_design" I take it? I think that is in sta, and I have avoided that for that reason. Is there a low overhead way to do that?

Reply:
Yes.  You could put a python API that still does a tcl eval internally but at least it would be more encapsulated in the right API.

Reply:
No, it;s in dbSta. What is your perspective on wrapping that?

Reply:
Looking at it in dbSta, it looks easy to pull out and put it on the Design object. Let me do that.

Reply:
Actually, it's already done, just need to update the regressions.